devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -248,12 +248,12 @@ def isSengled() {
 	device.getDataValue("manufacturer") == "sengled"
 }
 
-def configure() {
+def configure() {	
```

Comment:
this DTH should be left untouched.

Reply:
New PR here : https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/77964